content/en/docs/components/training/legacy-v1/_index.md

```diff
@@ -0,0 +1,11 @@
++++
+title = "Legacy (v1)"
+description = "Kubeflow Training V1 Documentation"
+weight = 999
++++
+
+{{% alert title="Old Version" color="warning" %}}
+This page is about **Kubeflow Training V1**, please see the [V2 documentation](/docs/components/training) for the latest information.
+
+Please follow [this guide for migrating to Kubeflow Training V2](/docs/components/training/admin-guides/migration)
+{{% /alert %}}
```

Comment:
Let me know if that message looks good to you @kubeflow/wg-training-leads @rimolive @varodrig @hbelmiro @StefanoFioravanzo.
If yes, I will add it to all Kubeflow Training V1 docs, similar to KFP: https://www.kubeflow.org/docs/components/pipelines/legacy-v1/overview/quickstart/